### PR TITLE
Update Elasticsearch mappings for entity indices

### PR DESCRIPTION
**Pull request by chiragjn**
Some mappings were found to be manually put outside of code which creates
a problem with new setups.

Fix a bunch of issues with `create` and `delete` index + aliases

### CODE REVIEW

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by srajdev** [comment]
:+1: No lint errors found.

File: datastore/elastic_search/create.py [hunk 1]
```diff
@@ -1,9 +1,15 @@
+import logging
```
Comment (srajdev) [inline datastore/elastic_search/create.py:1]: W1618 [34mimport missing `from __future__ import absolute_import`[0m

**Comment by chiragjn** [comment]
retest this please

**Comment by chiragjn** [comment]
retest this please

**Comment by haptik-deployment** [comment]
UNIT TESTS FAILED... Please resolve issues

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Pull request merged**